systemservices/marketplace/mesg.yml

```diff
@@ -0,0 +1,414 @@
+      sid: *inputSid
+      hash: &inputHash
+        name: ""
+        description: ""
+        type: String
+    outputs:
+      success:
+        name: ""
+        description: ""
+        data: *version
+      error: *outputError
+  serviceExist:
+    name: ""
+    description: ""
+    inputs:
+      sid: &inputSid
+        name: ""
+        description: ""
+        type: String
+    outputs:
+      success:
+        name: ""
+        description: ""
+        data:
+          exist:
+            name: ""
+            description: ""
+            type: Boolean
+      error: *outputError
+  createService:
+    name: ""
+    description: ""
+    inputs:
+      sid: *inputSid
+      from: &inputFrom
+        name: ""
+        description: ""
+        type: String
+      gas: &inputGas
+        name: ""
+        description: ""
+        type: Number
+        optional: true
+      gasPrice: &inputGasPrice
+        name: ""
+        description: ""
+        type: String
+        optional: true
+    outputs:
+      error: *outputError
+      success: &outputTransaction
+        data:
+          chainID:
+            name: "Chain ID"
+            description: "The ID of the chain the transaction is for. See https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md#list-of-chain-ids."
+            type: Number
+          nonce:
+            name: "Nonce"
+            description: "Nonce to use."
+            type: Number
+          to:
+            name: "To"
+            description: "The to address."
+            type: String
+          value:
+            name: "Value"
+            description: "The value in Wei."
+            type: String
+          gas:
+            name: "Gas"
+            description: "The maximum gas to use to execute this transaction"
+            type: Number
+          gasPrice:
+            name: "Gas price"
+            description: "The gas price in Wei."
+            type: String
+          data:
+            name: "Data"
+            description: "The data of the transaction."
+            type: String
+  createServiceOffer:
+    name: ""
+    description: ""
+    inputs:
+      sid: *inputSid
+      price:
+        name: ""
+        description: ""
+        type: String
+      duration:
+        name: ""
+        description: ""
+        type: String
+      from: *inputFrom
+      gas: *inputGas
+      gasPrice: *inputGasPrice
+    outputs:
+      error: *outputError
+      success: *outputTransaction
+  createServiceVersion:
+    name: ""
+    description: ""
+    inputs:
+      sid: *inputSid
+      hash:
+        name: ""
+        description: ""
+        type: String
+      manifest:
+        name: ""
+        description: ""
+        type: String
+      manifestProtocol:
+        name: ""
+        description: ""
+        type: String
+      from: *inputFrom
+      gas: *inputGas
+      gasPrice: *inputGasPrice
+    outputs:
+      error: *outputError
+      success: *outputTransaction
+  disableServiceOffer:
```

Comment:
I suggest to remove this. This can be done directly on the markeplace website. Keep it simple :)